packages/openssl/build.sh

```diff
@@ -1,10 +1,10 @@
 TERMUX_PKG_HOMEPAGE=https://www.openssl.org/
 TERMUX_PKG_DESCRIPTION="Library implementing the SSL and TLS protocols as well as general purpose cryptography functions"
 TERMUX_PKG_LICENSE="Apache-2.0"
-TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1:3.4.1
+TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
```

Comment:
Is there a specific reason you are changing the maintainer here, especially considering this seems to be your first commit for the package?

Reply:
It's been over 2 months since OpenSSL 3.5.0 has been released.
Since it doesn't have auto updates enabled (due to being a critical dependency for `apt`/`pacman`) it makes sense to have a maintainer assigned to get updates tested in a more timely manner.

I wasn't aware we hadn't updated it until `sslscan` failed to update yesterday.

Reply:
Unless I am missing something, how would changing the maintainer get you notified of upstream updates faster? That's seems more of a repology tracking issue by the whole team if auto updates are not enabled. Specific maintainers should normally only be assigned if they are authors, porters and/or someone who primarily maintains a package. If multiple Termux team members have a history of maintaining a package and responding to its issues, then maintainer should remain `@termux`. You would also automatically get notified in pulls for changes to such packages as default code owner.

Reply:
Makes sense to me I'll change it back to `@termux`

Reply:
Thanks for understanding. If you want to get notified faster, maybe there can be a weekly workflow run for (select) packages that don't have auto update enabled that generates a list of out of date packages, and opens an issue or something.

Reply:
Great.